eventing/README.md

```diff
@@ -70,6 +70,13 @@ Knative Eventing currently requires Knative Serving and Istio version 1.0 or
 later installed.
 [Follow the instructions to install on the platform of your choice](../install/README.md).
 
+### Configuring outbound network access
+
+Many of the sources require making outbound connections to create the event subscription,
+and if you have any functions that make use of any external (to cluster) services, you
+must enable it also for them to work.
+[Follow the instructions to configure outbound network access](../serving/outbound-network-access.md).
+
```

Comment:
You need a new sub-head here, or it looks like this is still about configuring outbound network access.

Reply:
Alternately, you can move this to right before "This document will be updated as additional sources ..."

Reply:
I just got rid of the subheader. originally I just wanted to add bold, but our linter didn't like that. I just removed it and added it as a paragraph.